templates/ts-apollo-mongodb-datasync-backend/README.md

```diff
@@ -1,45 +1,18 @@
-## Graphback Apollo+MongoDB Data Synchronization Runtime Template
+## Graphback MongoDB with Datasync + Apollo Server Template
 
-Template showcases Graphback Data Synchronization capabilities using
-resolver layer created in-memory as opposed to codegeneration.
+A template that provides you with an easy setup for your application's backend using MongoDB and Apollo server with TypeScript.
```

Comment:
Description here mentions nothing of Data Sync capabilities of this template - would be nice to have a line showcasing this.